rest-api/src/main/java/gov/cms/qpp/conversion/api/security/JwtAuthorizationFilter.java

```diff
@@ -0,0 +1,101 @@
+package gov.cms.qpp.conversion.api.security;
+
+import com.google.common.collect.Sets;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused import 'com.google.common.collect.Sets'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AUselessImportCheck)